### DISCUSSION TITLE
Some ideas

### DISCUSSION THREAD

**Discussion by Echeyak** [ValveSoftware/Dota2-Gameplay#5645] [Ideas]
1) The Behavior score system needs to be more strict in the lower MMR brackets, not all MMR are equal when it comes to behavior, at 3k MMR the behavior score system works well but at 2k MMR and lower it does not 10k behavior score games are most of the time full of toxicity.

2) I see most of the players vastly underperform when it comes to supporting roles, from my understanding this happens because they play 1 sup game every 4 core games just for the queue tokens and they don't really care if they win or lose for that one match, my suggestion is to reward 2 core queue tokens instead of 4 when the support player loses, this change will not only make support players better because they practice and tryhard more it will also lower queue times for core roles.

3) When you lose a match and immediately queue again you should never get matched up with the same teammates that you just lost with because when this happens it breads toxicity and Dota 2 already has enough of it. I know there is already a similar system in Dota plus but it is not the same first we cant use Dota plus to avoid after every single loss it becomes a hassle, second it is not necessary to block someone forever everyone has a bad day and by the next time you meet them again you will not even remember them and if someone is supper toxic and you really want to avoid them then you use Dota Plus, I believe this change will add a lot of value to the game.

4) MMR should be rewarded more fairly. My idea is to have a +-20 MMR for a win or loss then another +- 1 to 10 MMR rewarded based on the player's performance during the match, there is already a system in place that can tell how well a player performed so why not connect it with ranking for more fair ratings? This change will not only take away some toxicity from the player pool but it will also teach 
the player how to become better at the game.

5) When you are maxed out on ranked queue tokens(60) and also have all roles selected you should get queued on a role rotation instead of always getting the most in-demand role and this will consume one of your overflow queue tokens, this has potential to lower core queue times if a lot of people use this kind of queue(2sup3core instead of 1sup4core), you can already do that but its a hassle to change roles after every single match that's why people stick to one role even when they are comfortable playing all roles.

**Comment by kylogenetictree** [comment]
The MMR bit is just justifiable[.](https://sellerdash.pk/) I think the best of the ideas that I have seen so far. Great.
